docs/admin-guide/audit/audit.md

```diff
@@ -150,3 +156,21 @@ nodes:
  
     ...
 </pre>
+
+### vSphere Clusters
+
+Adding the following debug keys to the platform config can grant access to the primary master
+VM to allow debugging `kubeadm init` config issues:
+
+```YAML
+debug:
```

Comment:
What was the thinking behind this? How does this differ to just using `commands:` ?

Reply:
using the `master.commands` key is a more elegant approach. switching to it.